CHANGELOG.md

```diff
@@ -3,6 +3,91 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.2.0
+
+_Dec 9, 2021_
+
+A big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+### `@mui/x-data-grid@v5.2.0` / `@mui/x-data-grid-pro@v5.2.0`
+
+- 🚀 Introduce the [column pinning](https://mui.com/components/data-grid/columns/#column-pinning) feature (#2946) @m4theushw
+
+  <img src="https://user-images.githubusercontent.com/42154031/145425635-b6314fbe-2f1e-4b73-908f-33ee1fda20c7.gif" width="964" height="657">
```

Comment:
There are two problems with this image. First, by setting the width and height, we prevent the image to always respecting its ratio when the container is resized. It's broken on mobile:

<img width="372" alt="Screenshot 2021-12-09 at 20 57 58" src="https://user-images.githubusercontent.com/3165635/145466922-fc46cb77-938b-45d8-a7cc-6d178b72854a.png">

```suggestion
  <img src="https://user-images.githubusercontent.com/42154031/145425635-b6314fbe-2f1e-4b73-908f-33ee1fda20c7.gif" width="964">
```

---

Second, the image proportion doesn't seem at scale 1:1. I assume that recording was taken on https://mui.com/components/data-grid/demo/. The first column is 200px in the screen recording, while in reality, it's 160px wide. So, in the end, I believe the "right" configuration is:

```suggestion
  <img src="https://user-images.githubusercontent.com/42154031/145425635-b6314fbe-2f1e-4b73-908f-33ee1fda20c7.gif" width="771">
```

I have updated https://github.com/mui-org/material-ui-x/releases/tag/v5.2.0.

I think that developers based their judgment based on what they can see and extrapolate to the rest, so we have to give the changelog some love 😁

Reply:
Side note, flag emoji are not working on Windows. We fixed this problem in the Autocomplete demo of the core, we could consider the same patch here.

<img width="319" alt="Screenshot 2021-12-09 at 21 09 44" src="https://user-images.githubusercontent.com/3165635/145468402-797bccbb-40d1-4a94-85b1-f66aebbec3e4.png">

Reply:
Otherwise, great to see this recording :D